test/fixtures/module-config-section.json

```diff
@@ -0,0 +1,257 @@
+          }
+        ],
+        "warning": "Warning: This data-set is unpublished. Data may be subject to change or be inaccurate."
+      }
+    },
+    {
+      "moduleConfig": {
+        "info": ["Data source: Google Analytics"],
+        "value-attribute": "avgSessionDuration:sum",
+        "description": "",
+        "module-type": "single_timeseries",
+        "title": "Average time to complete a booking",
+        "axes": {
+          "y": [
+            {
+              "label": "Average session time",
+              "format": {
+                "type": "duration",
+                "unit": "m"
+              },
+              "key": "avgSessionDuration:sum"
+            }
+          ],
+          "x": {
+            "format": "date",
+            "key": [
+              "_start_at",
+              "_end_at"
+            ],
+            "label": "Date"
+          }
+        },
+        "modules": [],
+        "format-options": {
+          "type": "duration",
+          "unit": "m"
+        },
+        "slug": "time-to-complete-booking",
+        "data-source": {
+          "data-group": "driving-test-practical-public",
+          "data-type": "time-to-complete-by-event",
+          "query-params": {
+            "duration": 52,
+            "collect": ["avgSessionDuration:sum"],
+            "period": "week",
+            "filter_by": ["eventAction:booking-complete"],
+            "flatten": true
+          }
+        }
+      },
+      "options": {"backdrop": "https://www.performance.service.gov.uk/"},
+      "axes": {
+        "y": [
+          {
+            "label": "Average session time",
+            "format": {
+              "type": "duration",
+              "unit": "m"
+            },
+            "key": "avgSessionDuration:sum"
+          }
+        ],
+        "x": {
+          "format": "date",
+          "key": [
+            "_start_at",
+            "_end_at"
+          ],
+          "label": "Date"
+        }
+      },
+      "dataSource": {
+        "options": {
+          "json": true,
+          "backdrop": "https://www.performance.service.gov.uk/",
+          "url": "https://www.performance.service.gov.uk/data/driving-test-practical-public/t…on%3Asum&period=week&filter_by=eventAction%3Abooking-complete&flatten=true"
+        },
+        "data": [
+          {
+            "_count": 1,
+            "_end_at": "2015-02-09T00:00:00+00:00",
+            "_start_at": "2015-02-02T00:00:00+00:00",
+            "avgSessionDuration:sum": 1026424.4538161524
+          },
+          {
+            "_count": 1,
+            "_end_at": "2015-02-16T00:00:00+00:00",
+            "_start_at": "2015-02-09T00:00:00+00:00",
+            "avgSessionDuration:sum": 1003774.4788787357
+          }
+        ],
+        "warning": "Warning: This data-set is unpublished. Data may be subject to change or be inaccurate."
+      }
+    }
+  ],
+  "options": {"backdrop": "https://www.performance.service.gov.uk/"}
+}
```

Comment:
Can you add a newline at the end of this file and the other one you have added? Can you set your editor to add newlines at the end of files?